libsolidity/formal/CHC.cpp

```diff
@@ -1502,7 +1517,23 @@ optional<string> CHC::generateCounterexample(CHCSolverInterface::CexGraph const&
 				path.emplace_back("State: " + modelMsg);
 		}
 
-		path.emplace_back(txCex);
+		string txCex = summaryPredicate->formatSummaryCall(summaryArgs);
+
+		list<string> calls;
+		auto dfs = [&](unsigned node, unsigned depth, auto&& _dfs) -> void {
```

Comment:
Why did you need `_dfs` argument here?

Reply:
Because the recursive lambda won't be well typed unless you do that, afaik. I might also be missing something.